recipes/pytorchinfo/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "pytorchinfo" %}
+{% set version = "1.8.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/53/d9/2b811d1c0812e9ef23e6cf2dbe022becbe6c5ab065e33fd80ee05c0cd996/torchinfo-1.8.0.tar.gz
+  # https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/torchinfo-{{ version }}.tar.gz
```

```suggestion
  https://pypi.io/packages/source/t/torchinfo/torchinfo-{{ version }}.tar.gz
```

The jinja stuff is "cute" but it is overly complicated. lets just write down things that won't change and let the bot work!